### PR TITLE
Statistics about riders

### LINKED ISSUE #68: As a rider manager I want to the total number of riders associated and how many joined in the last 24 hours so that I can actively be involved in the well-being of my employees.

### LINKED ISSUE #70: As a rider manager I want to be able to see the average of the averages of every rider so that I can have an overall prespective of how all emplyees are being perceived by clients.

### LINKED ISSUE #71: As a rider manager I want to be able to see how many orders are in progress so I can study the busiest hours of the day and how the number fluctuates.

### PR DESCRIPTION
**Pull request by marianasps**
Closes #70 
Closes #68 
Closes #71 
Endpoint with information about:
- the average time it takes to deliver an order
- the average rating of the riders
- how many orders are yet to be delivered

### CODE REVIEW

File: projGenericService/src/main/java/ua/tqs/deliveryservice/repository/RiderRepository.java [hunk 1]
```diff
@@ -5,14 +5,17 @@
...
 import ua.tqs.deliveryservice.model.Rider;
 
-import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface RiderRepository extends JpaRepository<Rider, Long> {
     Optional<Rider> findByEmail(String email);
 
+    /*
```
Comment (hugofpaiva) [inline projGenericService/src/main/java/ua/tqs/deliveryservice/repository/RiderRepository.java:14]: Acho que já podes tirar isto

File: projGenericService/src/main/java/ua/tqs/deliveryservice/services/ManagerService.java [hunk 2]
```diff
@@ -29,6 +29,9 @@
     @Autowired
     private JwtUserDetailsService jwtUserDetailsService;
 
+    @Autowired
+    private PurchaseRepository purchaseRepository;
+
     // int countPurchaseByStatus(Status s);
```
Comment (hugofpaiva) [inline projGenericService/src/main/java/ua/tqs/deliveryservice/services/ManagerService.java:35]: Tbm penso que podes tirar isto

**Review by hugofpaiva — APPROVED** [review]

File: projGenericService/src/main/java/ua/tqs/deliveryservice/services/ManagerService.java (hunk 2 above)
Comment (carolinaaraujo00) [inline projGenericService/src/main/java/ua/tqs/deliveryservice/services/ManagerService.java:35]: comentários ficaram no PR

File: projGenericService/src/test/java/ua/tqs/deliveryservice/repository/RiderRepositoryTests.java [hunk 3]
```diff
@@ -157,16 +150,13 @@ public void testWhenGetSumReviewsAndQuantity_givenReviews_thenReturnSums() {
...
-        assertThat(res[0]).isEqualTo(33);
-        assertThat(res[1]).isEqualTo(11);
+        assertThat(res).isEqualTo((double) (32/10 + 1)/2);
     }
 
 
-
     /* -- helper -- */
     private Rider createAndSaveRider(int i) {
         Rider r = new Rider("rider"+i, "pwddddddd"+i, "rider"+i+"@email.com");
```
Comment (carolinaaraujo00) [inline projGenericService/src/test/java/ua/tqs/deliveryservice/repository/RiderRepositoryTests.java:162]: genial

**Review by carolinaaraujo00 — APPROVED** [review]

**Pull request merged**